hack/preview.sh

```diff
@@ -231,3 +264,9 @@ while :; do
   jq -r '.message' <<< "$STATE"
   sleep $INTERVAL
 done
+
+
+if [ -n $KEYCLOAK ] && [ -n $TOOLCHAIN ]; then
+  echo "Restarting toolchain registration service to pick up keycloak's certs."
+  oc delete deployment/registration-service -n toolchain-host-operator
```

Comment:
Probably will be necessary to wait until deployment is done before running any tests.

Reply:
Good point. I'll add wait here.